tests/test_execute.py

```diff
@@ -138,7 +138,7 @@ def test_check_passing_cmd_line_options(tmp_path):
     task_source = """
     import pytask
 
```

Comment:
I think this should be doubled up with a second number, different from 4. Just to avoid that whatever machine it is run on, it passes for reasons of the default happening to be 4.